src/components/NavBar/NavBar.js

```diff
@@ -6,7 +6,7 @@ const NavBar = () => (
   <header className="nav-container">
     <div className="nav-icon-and-h1">
       <img src={recycleIcon} className="recycle-icon" alt="recycle" />
-      <Link to="/">
+      <Link to="/recycle-me-ui">
```

Comment:
Again, in the future it would be nice to figure out how to get the routes working with just a /